src/tests/ftest/recovery/pass_1.py

```diff
@@ -0,0 +1,416 @@
+        should have rdb-pool.
+
+        Jira ID: DAOS-12029
+
+        :avocado: tags=all,full_regression
+        :avocado: tags=hw,medium
+        :avocado: tags=recovery,pass_1
+        :avocado: tags=test_lost_majority_ps_replicas
+        """
+        # 1. Create a pool with --nsvc=3.
+        self.pool = self.get_pool(svcn=3)
+
+        # 2. Stop servers.
+        dmg_command = self.get_dmg_command()
+        dmg_command.system_stop()
+
+        # 3. Remove /mnt/daos/<pool_uuid>/rdb-pool from rank 0 and 2.
+        rank_to_host = self.server_managers[0].ranks
+        host_0 = rank_to_host[0]
+        host_2 = rank_to_host[2]
+        hosts = NodeSet.fromlist([host_0, host_2])
+        command = f"sudo rm /mnt/daos0/{self.pool.uuid.lower()}/rdb-pool"
+        pcmd(hosts=hosts, command=command)
+
+        # 4. Start servers.
+        dmg_command.system_start()
+
+        # 5. Enable and start the checker.
+        dmg_command.check_enable()
+        # Calling dmg check start will automatically fix the issue with the default
+        # option, which is to retrieve the rdb-pool on any of the two ranks.
+        dmg_command.check_start()
+
+        # 6. Query the checker and verify the message
+        query_msg = ""
+        for _ in range(8):
+            check_query_out = dmg_command.check_query()
+            if check_query_out["response"]["status"] == "COMPLETED":
+                query_msg = check_query_out["response"]["reports"][0]["msg"]
+                break
+            time.sleep(5)
+
+        errors = []
+        if "corrupted pool without quorum" not in query_msg:
+            errors.append(
+                "Checker didn't detect orphan pool! msg = {}".format(query_msg))
+
+        # 7. Disable the checker.
+        dmg_command.check_disable()
+
+        # 8. Try creating a container. It should succeed.
+        cont_create_success = False
+        for _ in range(5):
+            time.sleep(5)
+            try:
+                self.container = self.get_container(pool=self.pool)
+                cont_create_success = True
+                break
+            except TestFail as error:
+                msg = (f"## Container create failed after running checker! "
+                       f"error = {error}")
+                self.log.debug(msg)
+
+        if not cont_create_success:
+            errors.append("Container create failed after running checker!")
+
+        # 9. Show that rdb-pool are recovered. i.e., at least three out of four ranks
+        # should have rdb-pool.
+        hosts = list(set(self.server_managers[0].ranks.values()))
+        count = 0
+        rdb_pool_path = f"/mnt/daos0/{self.pool.uuid.lower()}/rdb-pool"
+        for host in hosts:
+            node = NodeSet(host)
+            if check_file_exists(hosts=node, filename=rdb_pool_path):
+                count += 1
+                self.log.info("rdb-pool found at %s", str(node))
+
+        self.log.info("rdb-pool count = %d", count)
+        if count < 3:
```

Comment:
Just in case, could we avoid hardcoding?
```suggestion
        if count < len(hosts):
```

Reply:
Done